### PR TITLE
Update API versions

### PR DESCRIPTION
**Pull request by maciaszczykm**

### CODE REVIEW

**Comment by maciaszczykm** [comment]
/cc @shu-mutou 

Can you verify it?

**Comment by shu-mutou** [comment]
@maciaszczykm I do not have time this week. If no need to rush, I'll check this next week.

File: src/app/backend/scaling/scale.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ package scaling
 import (
 	"strconv"
 
+	"k8s.io/api/apps/v1"
```
Comment (shu-mutou) [inline src/app/backend/scaling/scale.go]: Please add "v1" at the first of this line as package alias.
We need to name for packages that are just version number like "v1"  explicitly.
Please see https://github.com/golang/go/issues/29041#issuecomment-443285973

**Comment by floreks** [comment]
Quickly tested overall functionalities and haven't found issues. Could you confirm @shu-mutou?

**Comment by shu-mutou** [comment]
/lgtm
I also do not face problems for now.

**Pull request merged**